IndomitableIbises/Bangazon-API-Sprint-1
# Cr friday commits

## Link to Ticket

No Ticket 

## Description of Proposed Changes

- Created Model/View/Url/Serializer/Faker Data for Training Programs and Employee Programmings
- Added comments and authorship
- This will be a reference to last Friday's commits

## Steps to Test

Outline the steps to test

```sh
git fetch --all
git checkout CR-FridayCommits
```
1. code .
2. **Delete** existing db.sqlite3 file **and** make sure your server isn't running
3. Run ./django_data.sh API faker_factory for **PC**
or
django_data.sh API faker_factory for **MAC**
4. python manage.py runserver
5. Look over currently functioning models and check personalized fake data
6. Date does not auto-populate the start and end date field in Training models
7. Made adjustments to the order table, it now accepts all foreign keys and got rid of the temp property of name. Payment now defaults to null on orders table.
8. Min and Max attendees is adjusted in faker data (No min and max on table, couldn't figure that out)

## Impacted Areas in Application

List general components of the application that this PR will affect:

*models.py
*views.py
*serializers.py
*url.py
*faker_factory.py


## Mentions @username
@rafcevallos  @sirwin4 @jcarter0149 

## Definition of Done

1. The project must be fully documented. This includes the following:
   1. Complete README that documents the steps to install the code, how to install any dependencies, or system configuration needed.
   1. Every class must be documented with purpose, author, and methods.
   1. Every method must be documented with purpose and argument list - which itself must contain a short purpose for each argument.
1. The project must be able to run fully, and without errors, on each teammate's system.
1. Fulfills every requirement.
Every line of code has been peer reviewed.
1. For projects that require unit testing, core functionality must be identified and have at least one test for each
